apps/cyberstorm-remix/app/settings/user/Connections/Connections.tsx

```diff
@@ -35,7 +36,13 @@ export const PROVIDERS: ProvidersType = [
 ];
 
 export default function Connections() {
+  const disconnectingProviderRef = useRef<string | null>(null);
   const outletContext = useOutletContext() as OutletContextShape;
+  const revalidator = useRevalidator();
+
+  const connectedProviders =
+    outletContext.currentUser?.connections?.length ?? 0;
+  const onlyOneConnected = connectedProviders === 1;
```

Comment:
I think this approach might be too simplistic to a point of being a bug. Doesn't it disable all the toggles when user has only one connection, i.e. prevent user from adding a second social auth to the same account?

Reply:
The issue is now fixed.